salt/modules/rabbitmq.py

```diff
@@ -848,7 +848,7 @@ def list_policies(vhost="/", runas=None):
     return ret
 
 
-def set_policy(vhost, name, pattern, definition, priority=None, runas=None):
+def set_policy(vhost, name, pattern, definition, apply_to=None, priority=None, runas=None):
     '''
```

Comment:
@dnABic 
For backwards compatibility, I would suggest that `apply_to` would follow `priority`.